config/pmd.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+
+<ruleset name="Basic"
+    xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 https://pmd.sourceforge.io/ruleset_2_0_0.xsd">
+  <description>
+The Basic ruleset contains a collection of good practices which should be followed.
+  </description>
+
+    <!-- Rules, that have been moved into a category -->
+    <rule ref="category/java/errorprone.xml/AvoidBranchingStatementAsLastInLoop" deprecated="true" />
```

Comment:
This syntax adds warnings to `mvn verify`. Please use the following one instead:
```xml
<rule ref="category/java/errorprone.xml">
      <exclude name="AvoidBranchingStatementAsLastInLoop" deprecated="true" />
      <!-- and so on -->
</rule>
```

Reply:
Sure :+1: